src/codec/builtin/ListCNFixedSizeCodec.ts

```diff
@@ -18,37 +17,35 @@ export class ListCNFixedSizeCodec {
     ): T[] {
         const type = FixSizedTypesCodec.decodeByte(frame.content, 0);
         const count = FixSizedTypesCodec.decodeInt(frame.content, 1);
+        let position = ListCNFixedSizeCodec.HEADER_SIZE;
```

Comment:
Can we define those a little bit below on the `default` branch, since they are used there?

Reply:
that given an eslint warning that's why I moved them

Reply:
I think we can suppress the warning, it doesn't make sense to define something that won't be used